src/menu.js

```diff
@@ -0,0 +1,116 @@
+const menuPage = () => {
+  const mainMenuDiv = document.createElement('div');
+  mainMenuDiv.setAttribute('class', 'hide-menu');
+  mainMenuDiv.setAttribute('id', 'main-menu-div');
+
+  const menu = document.createElement('div');
+  menu.setAttribute('id', 'main-menu');
+
+  mainMenuDiv.appendChild(menu);
+
+  // const menuHeader = document.createElement('h2');
+  // menuHeader.innerHTML = 'Our Menu';
+  //
+  // menu.appendChild(menuHeader);
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
noted